tests/Platforms/SQLServerPlatformTestCase.php

```diff
@@ -1224,7 +1228,7 @@ public static function getGeneratesIdentifierNamesInAlterTableSQL(): iterable
             // Quoted identifiers non-reserved keywords.
             [
                 new TableDiff(
-                    '`mytable`',
+                    new Table('mytable'),
```

Comment:
Is this supposed to keep the `?
```suggestion
                    new Table('`mytable`'),
```

Reply:
Thanks! The quotes got lost during refactoring.